docs/UserGuide.md

```diff
@@ -223,6 +225,11 @@ Examples:
 - `view_leave m/10` displays all employees that are taking leave in October
 - `view_leave m/10 d/IT` displays all employees in the IT department that are taking leave in October.
 
+### Resetting all employees' leaves : `reset_leaves`
+
+Sets the months in which employees have taken leaves back to zero.
```

Comment:
Perhaps can have a better description rather than set months to zero. Maybe can say reset all employees to have no recorded leaves, or something better.

Reply:
Done g!